### PR TITLE
User profile basic controller + "/" router + minor model changes

### PR DESCRIPTION
**Pull request by MasterCoookie**

### CODE REVIEW

**Review by wojciechsi — APPROVED** [review]
Dla mnie bomba

File: controllers/profileController.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const Applicant = require('../models/apllicantModel');
```
Comment (macibol775) [inline controllers/profileController.js:1]: typo

Comment (MasterCoookie) [reply controllers/profileController.js:1]: where?

File: models/apllicantModel.js
Comment (macibol775) [inline models/apllicantModel.js:1]: typo in file name

File: controllers/profileController.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+const profile_post = async (req, res) => {
+    // get user id
+    // TODO read from session
+    const _id = '64397e2fbed0bea2e17824d2';
+    const { name, surname, email, phoneNumber, birthDate, homeAdress, links } = req.body;
```
Comment (macibol775) [inline controllers/profileController.js:7]: address - typo

**Review by tomazaw — APPROVED** [review]

File: controllers/profileController.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    try {
+        await Applicant.findByIdAndUpdate(
+            _id,
+            { name, surname, email, phoneNumber, birthDate, homeAdress, links },
```
Comment (macibol775) [inline controllers/profileController.js:12]: same story

File: controllers/profileController.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        
+        // console.log(e);
+        res.status(400).json({ errors });
+    }
+}
+
+const profile_get = async (req, res) => {
+    try {
+        //TMP DUMMY DATA
+        const applicant = new Applicant({ name: 'dupa', surname: 'dupa2', email: 'pat.i.mat233@gmail.com', phoneNumber: '224444444', birthDate: '12.12.2001', homeAdress: 'kasztanowa 52 lipinki łużyckie', links: ['patrzuwa.ga', 'macibol.ga'] });
```
Comment (macibol775) [inline controllers/profileController.js:40]: as above

**Review by macibol775 — CHANGES_REQUESTED** [review]
correct awful typos

File: controllers/profileController.js (hunk 1 above)
Comment (macibol775) [reply controllers/profileController.js]: apllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllicant

**Review by macibol775 — APPROVED** [review]
please be consistent with semicolons

**Review by tomazaw — APPROVED** [review]

**Pull request merged**